src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/StegService.kt

```diff
@@ -185,6 +189,18 @@ class StegService(
     // Denne metoden kalles av HentStatusFraOppdragTask for å sende behandling videre etter OK status fra oppdrag er mottatt
     fun utførStegEtterIverksettelseAutomatisk(behandlingId: Long) {
         val behandling = behandlingRepository.hentAktivBehandling(behandlingId)
+
+        // opprett tilbakekreving task om det finnes en tilbakekrevingsvalg
+        tilbakekrevingRepository.findByBehandlingId(behandlingId)?.let {
+            if (it.valg == Tilbakekrevingsvalg.IGNORER_TILBAKEKREVING) {
+                logger.info(
+                    """Tilbakekrevingsvalg er ${it.valg.name} for behandling $behandlingId.
+                        Oppretter ikke tilbakekrevingsbehandling"""
+                )
+                return
```

Comment:
Vil denne returnen gå ut av metoden?
Vil man da ikke hoppe over stegene som utføres senere i denne metoden?

Reply:
@UyQuangNguyen Jeg tror denne returnen gå ut av metoden. Har lagt også en test metode for å teste dette

Reply:
Testen må isåfall sjekke at stegene utføres selv når valget er Tilbakekrevingsvalg.IGNORER_TILBAKEKREVING. I følge intellij så returner den ut av metoden:
<img width="1077" alt="Screenshot 2023-01-02 at 11 59 54" src="https://user-images.githubusercontent.com/110383605/210222681-66766186-02d5-4e66-a320-64005f371a08.png">

Reply:
Ja, fant dette. Fikser det nå

Reply:
fikset dette nå